cloudify_aws/route53/resources/hosted_zone.py

```diff
@@ -104,7 +108,7 @@ def create(ctx, iface, resource_config, **_):
     if iface.resource_id:
         params.update({'Name': iface.resource_id})
     if not params.get('CallerReference'):
-        params.update(dict(CallerReference=str(ctx.instance.id)))
+        params.update(dict(CallerReference=text_type(ctx.instance.id)))
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above